doc/intro/getting-started.rst

```diff
@@ -106,13 +106,13 @@ NAV will *not* autodiscover the routers and switches of your network. The
 assumption is that you already have this information in some inventory
 system.
 
-The *SeedDB* tool enables to you add and edit a multitude of "seed" information
+The *Seed Database* tool enables to you add and edit a multitude of "seed" information
 in the NAV database, which tells NAV what and how to monitor. The essential bit
 here is the :term:`IP Device`, which represents your switches, routers and
 other networked devices.
 
-The *SeedDB* tool is listed on NAV's *Toolbox* page, reachable from the grey
-navigation bar.
+The *Seed Database* tool is listed on NAV's *Tools* page, reachable from the navigation
+bar on top.
```

Comment:
The link may say "Tools", but the toolbox page is still the toolbox page (although I think people rarely visit it anymore, on account of the tools list being available as a dropdown on every page)